Fix #1 ; Fix #6 ; Feature 1,2,3 added and extra features

iosdev474 opened the pull request.

iosdev474 committed:
fix #1

fix #1

iosdev474 commented:
closing due to adding all 3 features in other pull request

iosdev474 commented:
please ignore commit messages :p and fix #1 , fix #6 and 
Added Feature 1,2,3
Added Features:
	Player should have different names
	Player should have different symbols
	Player's name can not be blank
	Player's symbol can not be blank
	Player can choose Alphabets,numbers etc as their symbol

nikilagar commented on TicTacToe/src/tictactoe/GameWindow.java:
```diff
@@ -26,18 +26,43 @@
     private final int PLAYINGAREAX = 110, PLAYINGAREAY = 100, CELLSIZE = 40, SPACEBETWEENCELLS = 10;
     private int currentPlayer = 0;
-    
+    int moved[][]=new int[3][3];
```
Why not boolean.

iosdev474 replied:
ok

nikilagar commented on TicTacToe/src/tictactoe/GameWindow.java:
```diff
@@ -46,10 +71,28 @@ private void endGame(int currentPlayer){
+                    draw=false;
+                }
+        }
+        if(draw==true)
```
No need to compare with true. If(draw) would suffice!!!

iosdev474 replied:
ok

nikilagar commented on TicTacToe/src/tictactoe/GameWindow.java:
```diff
@@ -150,32 +213,201 @@ public GameWindow() {
         pack();
     }// </editor-fold>//GEN-END:initComponents
 
+    private void nameTextfieldActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_nameTextfieldActionPerformed
```
Unnecessary function.

iosdev474 replied:
removed

nikilagar commented on TicTacToe/src/tictactoe/GameWindow.java:
```diff
@@ -150,32 +213,201 @@ public GameWindow() {
+    }//GEN-LAST:event_nameTextfieldActionPerformed
+
+    private void nextButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_nextButtonActionPerformed
+    if(!nameTextfield.getText().equals(""))
```
Indent the code.

iosdev474 replied:
done indenting

nikilagar commented on TicTacToe/src/tictactoe/GameWindow.java:
```diff
@@ -150,32 +213,201 @@ public GameWindow() {
+    }//GEN-LAST:event_nextButtonActionPerformed
+
+    private void startButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_startButtonActionPerformed
+    if(!nameTextfield.getText().equals(""))
```
Indent the code.

iosdev474 replied:
done indenting

urdarinda commented:
@nikilagar @iosdev474 My eyes hurt by seeing the title and the commit messages. Please fix :-1:

Commit:
Features #1,#3

Commit:
Features #1,#3

Commit:
Features #1,#3

Commit:
Features #1,#3

Commit:
Fix #1,Fix #6
Added Feature 1,2,3
Added Features:
	Player should have different names
	Player should have different symbols
	Player's name can not be blank
	Player's symbol can not be blank
	Player can choose Alphabets,numbers etc as their symbol

Commit:
gui edit only

iosdev474 commented:
adding another feature so closing request :-)

Commit:
corrected requested changes
